components/ws-manager/pkg/manager/manager.go

```diff
@@ -808,7 +816,7 @@ func (m *Manager) StopWorkspace(ctx context.Context, req *api.StopWorkspaceReque
 
 // stopWorkspace stops a workspace. If the workspace is already stopping, this is a noop
 func (m *Manager) stopWorkspace(ctx context.Context, workspaceID string, gracePeriod time.Duration) (err error) {
-	if m.Config.DryRun {
+	if m.Config.MaintenanceMode {
```

Comment:
What was dry run for? Just curious. :smile:

Reply:
That's a good question. To be honest I don't know.
But I checked the related code and did not find anywhere using the dryRun.

I did consider reusing the `dryRun` term to implement the maintenance mode.
However, it makes us confused that dryRun equals to maintenance mode.

Therefore, deprecating the dryRun and adding a new maintenanceMode configuration is reasonable.